archlinuxcn/clash-verge/PKGBUILD

```diff
@@ -32,7 +32,7 @@ source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/v${pkgver}.tar.gz
 )
 sha512sums=('9fe6676545d63768b156270d17e1bf75fef6c3d27202962faed5016d22fd237f6f223e7a917a8bd4fc6d200ea4db7614a687c930d478bdd02c1f7b0fb058ae6f'
             '2066dacf2e5e0135e6403cbfb825efcdf08bbcdc781407e6bb1fbb85143817b2b1abef641d20390ff7e5b3e91a509933e9eb17a64f9de7671445ac6d5363a44a'
-            'd1e8992cc0de168930002adccf7953416dfac199ea11886043de58b0df366e4337d5ec548fa3bf8be30575596a5e71dd31960db72f9e067ff131feb756bc96c1')
+            '82d7afef87af300eafea33c3956db636e8505501f206ddaf382017664c98567d72a88ae835c1e917ae73f4824cd3bd6acac24c1894a369c9289dafb100a2e6cb')
 options=(!lto)
 
 prepare() {
```

Comment:
```
==> Starting prepare()...
patching file src-tauri/src/main.rs
Hunk #1 FAILED at 106.
1 out of 1 hunk FAILED -- saving rejects to file src-tauri/src/main.rs.rej
==> ERROR: A failure occurred in prepare().
```

Reply:
Here is, the old patch worked with 1.4.10.
The new patch worked with 1.5.1(newest).
So here wrong because old clash-verge with new patch.
I once thinks, lilac may auto updated the new one, so i didn't change the pkgver.
I updated manually now.